spmd/tensor/ops/tensor_ops.py

```diff
@@ -15,10 +15,13 @@ def default_prop_rule(op_schema: OpSchema) -> OutputSharding:
 
 
 default_prop_ops = [
+    "aten._local_scalar_dense.default",
+    "aten.clone.default",
+    "aten.copy_.default",
+    "aten.detach.default",
     "aten.is_same_size.default",
     "aten.ones_like.default",
-    "aten.detach.default",
-    "aten.clone.default",
+    "aten.new_empty_strided.default",
```

Comment:
Can you also add "aten._to_copy.default",?

As in https://github.com/pytorch/tau/pull/382/files#diff-34b9e654af2d5e93ed673c8e416fe783e49d1e8a81b443f8d675725649429a77R22